pkg/sql/table.go

```diff
@@ -479,8 +535,15 @@ func (tc *TableCollection) addUncommittedTable(desc sqlbase.TableDescriptor) {
 	tc.releaseAllDescriptors()
 }
 
-func (tc *TableCollection) addUncommittedDatabase(name string, id sqlbase.ID, dropped bool) {
-	db := uncommittedDatabase{name: name, id: id, dropped: dropped}
+type dbAction bool
+
+const (
+	dbCreated dbAction = false
+	dbDropped dbAction = true
+)
+
+func (tc *TableCollection) addUncommittedDatabase(name string, id sqlbase.ID, action dbAction) {
+	db := uncommittedDatabase{name: name, id: id, dropped: action == dbDropped}
```

Comment:
I suppose you could store the action in uncommittedDatabase as opposed to the boolean.